index.html

```diff
@@ -931,6 +931,14 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
     <td><img src='https://img.shields.io/swagger/valid/2.0/https/bitbucket.org/api/swagger.json.svg?maxAge=2592000' alt=''/></td>
     <td><code>https://img.shields.io/swagger/valid/2.0/https/bitbucket.org/api/swagger.json.svg</code></td>
   </tr>
+  <tr><th> Uptime Robot status: </th>
+    <td><img src='https://img.shields.io/uptimerobot/status/u956-afus321g565fghr519.svg?maxAge=2592000' alt=''/></td>
```

Comment:
Agreed. Let's go a step further and reject the user keys, to reduce the chance that people share them by accident.